codebuild_specs/run_e2e_tests_linux.yml

```diff
@@ -3,7 +3,8 @@ env:
   shell: bash
   variables:
     CLI_REGION: us-east-1
-    TEST_SUITE: src/__tests__/auth_2a.test.ts|src/__tests__/auth_2b.test.ts|src/__tests__/auth_2d.test.ts|src/__tests__/auth_2f.test.ts
+    TEST_SUITE: src/__tests__/geo-add-b.test.ts
```

Comment:
I don't think you want to overwrite this line. It's important to run multiple tests here.